### PR TITLE
Added double direction

### PR DESCRIPTION
**Pull request by tanrax**

### CODE REVIEW

File: burn.el [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
 ;; URL: https://github.com/erickgnavar/burn.el
-;; Version: 0.1.0
+;; Version: 0.2.0
 ;; SPDX-License-Identifier: GNU General Public License v3.0 or later
 
 ;;; Commentary:
 ;; Replace the buffer content with 🔥.
 
 ;;; Code:
+(provide 'burn)
```
Comment (erickgnavar) [inline burn.el:13]: This should be at the end of the file

File: burn.el [hunk 2]
```diff
@@ -1,34 +1,45 @@
...
+    (replace-match burn--emoji)))
+
+(defun burn--replace-forward-at-point ()
+  "Replace the forward character at point with burn--emoji."
+  (when (re-search-forward (concat "[^ \t\n" burn--emoji "]") nil :noerr)
+    (replace-match burn--emoji)))
 
 (defun burn-code ()
-  "Replace code with 🔥 emoji."
+  "Replace code with burn--emoji."
```
Comment (erickgnavar) [inline burn.el:35]: This way `burn--emoji` will be highlighted so we can know is a variable that was defined previously

```suggestion
  "Replace code with `burn--emoji'."
```

File: burn.el [hunk 3]
```diff
@@ -1,34 +1,45 @@
...
   (interactive)
-  (while (< (line-number-at-pos) (line-number-at-pos (point-max)))
-    (progn
-      (beginning-of-line)
-      (burn--replace-line)
-      (forward-line))))
+  ;; Burn
```
Comment (erickgnavar) [inline burn.el:37]: this comment doesn't contribute much 🤔

File: burn.el [hunk 4]
```diff
@@ -1,34 +1,45 @@
...
+  (while (not (burn--is-empty))
+    (sit-for burn--interval)
+    (burn--replace-backward-at-point)
+    (burn--replace-forward-at-point))
+  (erase-buffer))
 
 ;;; burn.el ends here
+
```
Comment (erickgnavar) [inline burn.el:45]: unnecessary diff

File: burn.el [hunk 5]
```diff
@@ -1,34 +1,45 @@
...
-    (progn
-      (beginning-of-line)
-      (burn--replace-line)
-      (forward-line))))
+  ;; Burn
+  (while (not (burn--is-empty))
+    (sit-for burn--interval)
+    (burn--replace-backward-at-point)
+    (burn--replace-forward-at-point))
+  (erase-buffer))
```
Comment (erickgnavar) [inline burn.el:42]: why is this required? At this point all the content should be replaced with the emoji

File: README.md [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
+Empty the buffer with a hot 🔥fire🔥 .
 
 ![burn-code](./images/burn-code.gif)
 
 ## Install
 
-### Cloning the repo
-
-Clone this repo somewhere
+1. Cloning the repo o download `burn.el`.
```
Comment (erickgnavar) [inline README.md:11]:
```suggestion
1. Clone the repo o download `burn.el`.
```

File: burn.el (hunk 2 above)
Comment (tanrax) [reply burn.el:35]: ok

File: burn.el (hunk 4 above)
Comment (tanrax) [reply burn.el:45]: ok

File: burn.el (hunk 3 above)
Comment (tanrax) [reply burn.el:37]: ok

File: README.md (hunk 6 above)
Comment (tanrax) [reply README.md:11]: Clone the repo OR download...

File: burn.el (hunk 1 above)
Comment (tanrax) [reply burn.el:13]: of course
